src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,80 @@
+package mate.jdbc.model;
+
+import java.util.Objects;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
+
+    }
+
+    public Manufacturer(String name, String country) {
+        this.name = name;
+        this.country = country;
+    }
+
+    public Manufacturer(Long id, String name, String country) {
+        this.id = id;
+        this.name = name;
+        this.country = country;
+    }

```

Comment:
```suggestion
    public Manufacturer(Long id, String name, String country) {
        this.id = id;
        this(name, country)
    }

```

it's better to use existing constructor

Reply:
done

Reply:
done